Update 'What's mixed content' article (#4206)
GoogleChrome/web.dev: src/site/content/en/secure/what-is-mixed-content/index.md

an attacker can track users via mixed content requests.
The attacker can tell which pages a user visits and which products they view based on images or other resources that the browser loads.

If passive mixed content is present most browsers will indicate in the URL bar that the page is not secure,
even when the page itself was loaded over HTTPS.

> You can observe this behavior with this [Glitch demo](https://passive-mixed-content.glitch.me/)

Reviewer: Suggestion: remove `Glitch` since readers may not be familiar with the term.

Automatic upgrading means that if the asset is available over HTTPS, but has been hardcoded as HTTP,
the browser will load the HTTPS version. If no secure version can be found the asset will not load.

> Whenever it detects mixed content or auto-upgrades passive mixed content, Chrome logs detailed messages to the Issues Tab in Chrome DevTools
> to guide you on how to fix the specific issue.

Reviewer 2: ```suggestion
Whenever it detects mixed content or auto-upgrades passive mixed content,
Chrome logs detailed messages to the **Issues** tab in DevTools
to guide you on how to fix the specific issue.
```

<figure class="w-figure">
  <img class="w-screenshot"
      src="passive-mixed-content.jpg"

>       alt="The Issues tab in DevTools shows detailed information about the specific mixed content issue and how to fix it">

Reviewer: Suggestion: call it `Chrome DevTools` for clarity -- Or wait, since Edge has it too: what do you call it in this case? `Chromium DevTools`? Or simply `DevTools in Chrome and Edge` (although this over-simplifies)?

</figure>

> 

Reviewer: Suggestion/Nit (not a must for this PR since the original article is like that, but while we're at it): we could add a mention of what other browsers do, in order to make this web.dev article browser-agnostic.
E.g. `Firefox and Safari log messages in the developer console.`

most browsers already block this type of content by default to protect users,
but functionality varies between browser vendors and versions.

This other [Glitch demo](https://active-mixed-content.glitch.me/) contains examples of active mixed content.
[Load the example over HTTP](http://active-mixed-content.glitch.me/)
to see the content that's blocked when you [load the example over

> HTTPS](https://active-mixed-content.glitch.me/). Blocked content will also be detailed in the Issues Tab.

Reviewer: Suggestion/Nit (not a must for this PR since the original article is like that, but while we're at it): we could add a mention of what other browsers do, in order to make this web.dev article browser-agnostic.
E.g. `Browsers also highlight blocked content in developer tooling: block content issues are detailed in the Issues Tab in Chrome and Edge, and Firefox and Safari log messages in the developer console.`

Reviewer: Added below

<figure class="w-figure">
  <img class="w-screenshot"
      src="active-mixed-content.jpg"

>       alt="The Issues tab in DevTools shows detailed information about the specific mixed content issue and how to fix it">

Reviewer: Same comment as above regarding `DevTools`

most browsers already block this type of content by default to protect users,
but functionality varies between browser vendors and versions.

This other [demo](https://active-mixed-content.glitch.me/) contains examples of active mixed content.
[Load the example over HTTP](http://active-mixed-content.glitch.me/)
to see the content that's blocked when you [load the example over

> HTTPS](https://active-mixed-content.glitch.me/). Blocked content will also be detailed in the Issues Tab.

Reviewer 2: ```suggestion
HTTPS](https://active-mixed-content.glitch.me/). Blocked content will also be detailed in the Issues tab.
```

<figure class="w-figure">
  <img class="w-screenshot"
      src="active-mixed-content.jpg"
      alt="The Issues tab in Chrome DevTools shows detailed information about the specific mixed content issue and how to fix it">
</figure>

{% Aside %}

> Browsers also highlight blocked content in developer tooling. Block content issues are detailed in the Issues Tab in Chrome and Edge, and Firefox and Safari log messages in the developer console.

Reviewer: I made a typo here, I meant "blocked content issues", my mistake

> Browsers also highlight blocked content in developer tooling. Blocked content issues are detailed in the Issues Tab in Chrome and Edge, and Firefox and Safari log messages in the developer console.

Reviewer 2: ```suggestion
  Browsers also highlight blocked content in their DevTools. Blocked content
  issues are detailed in the **Issues** tab in Chromium-based browsers. Firefox and Safari
  log messages in the console.
```
